operator/src/main/java/org/keycloak/operator/controllers/KeycloakSharedCsvMetadata.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.operator.controllers;
+
+import io.quarkiverse.operatorsdk.bundle.runtime.CSVMetadata;
+import io.quarkiverse.operatorsdk.bundle.runtime.SharedCSVMetadata;
+
+@CSVMetadata(
+    version = "KCOP_NEXT",
+    name = "keycloak-operator.KCOP_NEXT",
+    replaces = "keycloak-operator.KCOP_PREVIOUS",
+    displayName = "Keycloak Operator",
+    provider = @CSVMetadata.Provider(
+        name = "Red Hat"
+    ),
+    maturity = "stable",
+    keywords = {
+        "Keycloak",
+        "Identity",
+        "Access"
+    },
+    maintainers = {
+        @CSVMetadata.Maintainer(
+            email = "keycloak-dev@lists.jboss.org",
```

Comment:
```suggestion
            email = "keycloak-dev@googlegroups.com",
```
Regarding the Keycloak website, the official way is via Google groups.

Reply:
I can update that for sure